MREUnityRuntime/MREUnityRuntimeLib/Core/Actor.cs

```diff
@@ -395,16 +406,21 @@ internal OperationResult EnableText(TextPatch textPatch)
             };
         }
 
+        private MWScaledTransform localTransform = new MWScaledTransform();
+        private MWTransform appTransform = new MWTransform();
```

Comment:
In this class, we have `AppTransform`, `_appTransform`, and `appTransform`. Can you differentiate them through renaming, or clarify their usage in comments?

Reply:
Standard naming conventions is __name_ for private member variables, and no underscore for "local" variables which is what these are replacing.  I will add a comment and I can do a double or triple underscore.  That should really make it clear not to use them elsewhere in the class.

Reply:
A custom naming convention for temporary vars that stick around is good; brings a nice increase in clarity. Let's use this convention everywhere (not just in Actor.cs). Minor nit: `__methodVar_` seems excessive, maybe `_tmp_`?

---
In reply to: [306007763](https://github.com/microsoft/mixed-reality-extension-unity/pull/90#discussion_r306007763) [](ancestors = 306007763)

Reply:
I think that `__methodVar_` is more descriptive than tmp.  I also thing the verbosity is good for quickly identifying the usage of the variable from intellisense.  If you feel strongly enough I could be convinced for `__tmp_`, though i still would insist on a double underscore than single.  It will filter them out as you type a member variable with a single underscore.

Reply:
`__methodVar_` it is.